src/schema.graphql

```diff
@@ -3276,16 +3329,34 @@ extend type PatronCharge {
   feeFine: FeeFine
 }
 
+# This is the same as FOLIO's FeeFineAction type, but it removes 1 field: 'source'.
+# 'source' contains the name of the staff member who performed the action and we don't want to expose that.
+type CustomFeeFineAction {
+  dateAction: DateTime
+  typeAction: String
+  comments: String
+  notify: Boolean
+  amountAction: Float
+  balance: Float
+  transactionInformation: String
+  createdAt: UUID
+  originalCreatedAt: String
+  paymentMethod: String
+  accountId: UUID!
+  userId: UUID!
+  id: UUID
+}
+
```

Comment:
What if we just deleted the key from the returned object in the resolver, rather than creating a new type? Would that work?

Reply:
I tried that first, but the "source" option still showed up in the Apollo UI even if it came back null. So I figured this was a more complete approach. Then I researched trying to do something like if you could designate a field to "ignore" in extending a graphql type, so that we could use the original type rather overwriting it completely like this. But that didn't exist. So this was where I landed.

Reply:
Do you think it's a problem for API consumers to know that there _is_ a source field? That seems fine to me, but I guess I don't have a complete picture of the security concerns around this.

Reply:
I guess I felt like if it will always be null, the least confusing option was to remove it entirely.

Reply:
I think as long as we document in a comment that it'll be nulled for security reasons (or even replace it with a string like `[redacted]`, that's preferable to duplicating all the fields from FOLIO's type. The only people who will ever use this API are us, and if we don't want to use that value, I think it's OK for it to be null. On the application side (mylibrary), we won't even bother having a field/variable, so it basically will be invisible.

Reply:
I agree with @thatbudakguy ; I don't think there's any need to suppress data in this API, is there? We're consuming this API, and its on our downstream apps not to expose data.

Reply:
@cbeer I just updated this to reflect this feedback.